Configuration/StandardSequences/python/AlCaHarvesting_cff.py

```diff
@@ -248,15 +248,15 @@
 
 # --------------------------------------------------------------------------------------
 # PPS calibration
-ALCAHARVESTPPSTimingCalibration_metadata = cms.PSet(record = cms.untracked.string('PPSTimingCalibrationRcd'))
+ALCAHARVESTPPSTimingCalibration_metadata = cms.PSet(record = cms.untracked.string('PPSTimingCalibrationRcd_HPTDC'))
```

Comment:
This is now mismatched with 
https://github.com/cms-sw/cmssw/blob/eb2d3a59a3aa2a722da88f97059810aa841d35c1/CalibPPS/TimingCalibration/plugins/PPSTimingCalibrationPCLHarvester.cc#L150